packages/superset-ui-core/test/query/buildQueryObject.test.ts

```diff
@@ -149,7 +151,8 @@ describe('buildQueryObject', () => {
       viz_type: 'table',
       timeseries_limit_metric: metric,
     });
-    expect(query.timeseries_limit_metric).toEqual(metric);
+    expect(query.orderby).toEqual([[metric, true]]);
+    expect(query.timeseries_limit_metric).toEqual(undefined);
```

Comment:
IMO this test should sort primarily by `granularity_sqla`, not `metric`. So something like 
```javascript
expect(query.orderby).toEqual([['ds, true], [metric, true]]);
```